domain/src/main/java/woowacourse/movie/theater/Theater.kt

```diff
@@ -0,0 +1,34 @@
+package woowacourse.movie.theater
+
+import woowacourse.movie.ticket.Position
+import woowacourse.movie.ticket.Seat
+import woowacourse.movie.ticket.SeatRank
+
+data class Theater(
+    val id: Long,
+    val rowSize: Int,
+    val columnSize: Int,
+    val sRankRange: List<IntRange>,
+    val aRankRange: List<IntRange>,
+    val bRankRange: List<IntRange>,
+) {
+    private val rankMap: Map<SeatRank, List<IntRange>>
+        get() = mapOf(
+            SeatRank.S to sRankRange,
+            SeatRank.A to aRankRange,
+            SeatRank.B to bRankRange,
+        )
+
+    fun selectSeat(position: Position): Seat {
+        return Seat(getSeatRank(position.row), position)
+    }
+
+    private fun getSeatRank(row: Int): SeatRank {
+        rankMap.entries.forEach {
```

Comment:
```suggestion
        rankMap.forEach { k, v ->
```
map 은 entry 를 순회할 수 있는  forEach 도 제공하고 있으니 참고 부탁드려요~!